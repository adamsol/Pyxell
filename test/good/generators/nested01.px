
func f(step: Int = 1): Void def
    func* between(a: Int, b: Int): Int def
        for x in a..b by step do
            yield x

    print [...between(4, 7)]

f()
f(3)
f(10)
