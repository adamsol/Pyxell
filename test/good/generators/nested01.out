[4, 5, 6, 7]
[4, 7]
[4]
